docs/getting-started/new_project_guide.md

```diff
@@ -310,7 +313,8 @@ fuzz targets get to the code you expect:
 
     ```bash
     $ python infra/helper.py build_fuzzers --sanitizer coverage $PROJECT_NAME
-    $ python infra/helper.py coverage $PROJECT_NAME <fuzz_target>
+    $ python infra/helper.py run_fuzzer $PROJECT_NAME <fuzz_target>
```

Comment:
This is all stale, sorry about.
The right way is not to do run_fuzzer, but provide corpus from a local dir. This page wasn't updated, see updated docs/advanced-topics/code_coverage.md
it should be one command:
python infra/helper.py coverage --fuzz-target=<fuzz_target> \
    --corpus-dir=<my_local_corpus_dir> $PROJECT_NAME

See if this works for you. i think you dont need --no-corpus-download when you provide --corpus-dir.

@oliverchang as well.

Reply:
This makes sense. So for local code coverage sanity check, it is actually get the coverage of specified corpus. And for local test, we just provide the corpus to see coverage, otherwise, it will download from the cloud.
Let me change this PR by redircting to the advanced topic pages.